diff --git a/admin/sql/CreateFKConstraints.sql b/admin/sql/CreateFKConstraints.sql
index d6e1acd..fda3931 100644
--- a/admin/sql/CreateFKConstraints.sql
+++ b/admin/sql/CreateFKConstraints.sql
@@ -1,10 +1,10 @@
 -- Automatically generated, do not edit.
 \set ON_ERROR_STOP 1
 
-ALTER TABLE annotation
-   ADD CONSTRAINT annotation_fk_editor
-   FOREIGN KEY (editor)
-   REFERENCES editor(id);
+--ALTER TABLE annotation
+--   ADD CONSTRAINT annotation_fk_editor
+--   FOREIGN KEY (editor)
+--   REFERENCES editor(id);
 
 ALTER TABLE artist
    ADD CONSTRAINT artist_fk_name
diff --git a/admin/sql/CreateIndexes.sql b/admin/sql/CreateIndexes.sql
index a5977bc..ae5cbc8 100644
--- a/admin/sql/CreateIndexes.sql
+++ b/admin/sql/CreateIndexes.sql
@@ -198,11 +198,11 @@ CREATE INDEX artist_name_idx_lower_name ON artist_name (lower(name));
 CREATE INDEX label_name_idx_lower_name ON label_name (lower(name));
 
 -- musicbrainz_collate indexes for unicode sorting
-CREATE INDEX release_name_idx_musicbrainz_collate ON release_name (musicbrainz_collate(name));
-CREATE INDEX artist_name_idx_musicbrainz_collate ON artist_name (musicbrainz_collate(name));
-CREATE INDEX label_name_idx_musicbrainz_collate ON label_name (musicbrainz_collate(name));
-CREATE INDEX track_name_idx_musicbrainz_collate ON track_name (musicbrainz_collate(name));
-CREATE INDEX work_name_idx_musicbrainz_collate ON work_name (musicbrainz_collate(name));
+--CREATE INDEX release_name_idx_musicbrainz_collate ON release_name (musicbrainz_collate(name));
+--CREATE INDEX artist_name_idx_musicbrainz_collate ON artist_name (musicbrainz_collate(name));
+--CREATE INDEX label_name_idx_musicbrainz_collate ON label_name (musicbrainz_collate(name));
+--CREATE INDEX track_name_idx_musicbrainz_collate ON track_name (musicbrainz_collate(name));
+--CREATE INDEX work_name_idx_musicbrainz_collate ON work_name (musicbrainz_collate(name));
 
 
 
